packages/reactive_ble_mobile/protos/bledata.proto

```diff
@@ -13,9 +13,10 @@ message DeviceScanInfo {
     string name = 2;
     GenericFailure failure = 3;
     repeated ServiceDataEntry serviceData = 4;
-    bytes manufacturerData = 6;
-    repeated Uuid serviceUuids = 7;
+    bytes manufacturerData = 7;
+    repeated Uuid serviceUuids = 8;
     int32 rssi = 5;
+    IsConnectable isConnectable = 6;
```

Comment:
Not sure if we need to maintain compatibility with the older versions here, but, either way, reshuffling numbers like this makes no sense. The order is both weird _and_ not compatible with the previous version.

While it could make sense to stay backwards compatible, in theory, I don't think it is important for this project practically.

Please, choose one or the other approach and either don't change the old numbers, or put them all in order.